docs/section21/Rule21-5.md

```diff
@@ -0,0 +1,77 @@
+
+2. B-RMR is modeled with at least one air-side system that is Type-1a, 7, 11, 12 or that is Type-1, 5, 7, 11, 12:
+
+  - If P-RMR is not modeled with purchased cooling: `if NOT purchased_chw_hhw_status_dict["PURCHASED_COOLING"]:`
+
+    - Check if B-RMR is modeled with at least one air-side system that is Type-1, 5, 7, 11 or 12, continue to rule logic: `PLACEHOLDER`
+
+    - Else, rule is not applicable: `else: rule_applicability_flag = FALSE`
+
+  - Else, P-RMR is modeled with purchased cooling: `else:`
+
+    - Check if B-RMR is modeled with at least one air-side system that is Type-1a, 7, 11 or 12, continue to rule logic: `PLACEHOLDER`
+
+    - Else, rule is not applicable: `else: rule_applicability_flag = FALSE`
+
+## Rule Logic:  
+
+- Get zone conditioning category dictionary for B_RMR: `zone_conditioning_category_dict = get_zone_conditioning_category(B_RMR)`
+
+- Get dictionary that saves the list of zones and the total floor area served by each CHW or HHW loop: `loop_zone_list_w_area_dict = get_loop_zone_list_w_area(B_RMR)`
+
+- For each fluid loop in B_RMR: `for fluid_loop_b in B_RMR.ASHRAE229.fluid_loops:`
+
+  - Check if fluid loop is heating type: `if fluid_loop_b.type == "HEATING":`
+
+    - Add to the list of zones with their total floor area served by heating loop: `heating_loop_zone_list.append(loop_zone_list_w_area_dict[fluid_loop_b.id]["ZONE_LIST"]), heating_loop_conditioned_zone_area += loop_zone_list_w_area_dict[fluid_loop_b.id]["TOTAL_AREA"]`
+
+- For each zone in B_RMR: `for zone_b in B_RMR...zones:`
+
+  - Check if zone is classified as conditioned (including directly and indirectly conditioned) and zone is not connected to any heating loop: `if ( zone_conditioning_category_dict[zone_b.id] in ["CONDITIONED RESIDENTIAL", "CONDITIONED NON-RESIDENTIAL", "CONDITIONED MIXED"] ) AND ( NOT zone_b in heating_loop_zone_list ):`
+
+    - Classify zone as indirectly conditioned and add zone total area to total area served by heating loop: `heating_loop_conditioned_zone_area += SUM(space.floor_area for space in zone.spaces)` (Note XC, in this logic zone might be adjacent to directly conditioned zones that are cooling only, but its area will still be added to HHW loop)
+
+**Rule Assertion:**
+
+- Case 1: If baseline building design plant serves a conditioned floor area of 15,000sq.ft. or less, and if only one boiler is modeled in B_RMR: `if ( heating_loop_conditioned_zone_area <= 15000 ) AND ( LEN(B_RMR.ASHRAE229.boilers) == 1 ): PASS`
+
+- Case 2: Else if baseline building design plant serves a conditioned floor area more than 15,000sq.ft. and two boilers are modeled in B_RMR and the two boilers are sized equally: `else if ( heating_loop_conditioned_zone_area > 15000 ) AND ( LEN(B_RMR.ASHRAE229.boilers) == 2 ) AND ( B_RMR.ASHRAE229.boilers[0].rated_capacity == B_RMR.ASHRAE229.boilers[1].rated_capacity ): PASS`
+
+- Case 3: Else: `else: FAIL`
+
```

Comment:
Should we add Case 4 with NOT APPLICALBE outcome?

Reply:
As discussed on call, we will skip NOT APPLICABLE outcome in RDS.